dist/pathfora.js

```diff
@@ -562,7 +626,7 @@
      */
     parseQuery: function (url) {
       var query = {};
-      var pieces = url.split('?');
+      var pieces = utils.escapeURI(url, { keepEscaped: true }).split('?');
```

Comment:
Yeah, you were totally right, of course. I updated the PR. I think I got it right this time.